src/syntax_highlight.rs

```diff
@@ -19,7 +20,14 @@ use engarde::Raw as Highlight;
 use engarde::Syntax as Highlight;
 
 lazy_static! {
-    static ref HIGHLIGHT: Highlight = Highlight::new();
+    static ref HIGHLIGHT: Highlight = {
+        let mut highlight = Highlight::new();
+        let syntaxes_path = Path::new("./_syntaxes");
+        if syntaxes_path.exists() {
+            highlight.load_custom_syntaxes(syntaxes_path);
+        }
+        highlight
+    };
```

Comment:
We're going to need to create this at runtime so we can load the correct root for `_syntaxes`.

Could you do that refactor in a separate PR?

Reply:
Yeah, I'm happy to help out in any way to get this landed, but I'm not quite sure what you mean. I thought `lazy_static` would initialize this at runtime
> Using this macro, it is possible to have statics that require code to be executed at runtime in order to be initialized.

I could use a little more direction for what you're asking.

Reply:
I went ahead and did it real quick: https://github.com/cobalt-org/cobalt.rs/pull/1052

Reply:
Merged